e2e/run_tests.js

```diff
@@ -128,4 +130,66 @@ async function runTests() {
     process.exit(failedTests); // exit with the number of failed tests
 }
 
+const result = [
+    {status: 'Passed', priority: 'none', cutOff: 100, color: '#43A047'},
+    {status: 'Failed', priority: 'low', cutOff: 98, color: '#FFEB3B'},
+    {status: 'Failed', priority: 'medium', cutOff: 95, color: '#FF9800'},
+    {status: 'Failed', priority: 'high', cutOff: 0, color: '#F44336'},
+];
```

Comment:
Feel free to comment.  We can discuss more about this, especially the priority.  My proposal is to add indication on support level (weekly rotation) so that failing tests are attended/verified on time with the good amount of support man hours.  
- `status` - all are failed except when 100% passed
- `cutOff` is measured in terms of passing rate but another option is the number of failing tests (5, 10,  15, more than 15)
- `priority` - to categorize support level needed.  Such could be
  - `none` and `low` could be handled by the support person in-charge for the week
  - `medium` - should add one more support (total of 2 support to verify failing tests) 
  - `high` - should require all support to verify failing tests